### PR TITLE
FLOE-420: Serve the first discovery tool

### PR DESCRIPTION
**Pull request by jobara**
Using gpii-express to serve an instance of the first discovery tool ( fetched via NPM ).

vagrant and other provisioning files have been provided to simplify the development setup.

https://issues.fluidproject.org/browse/FLOE-420

### CODE REVIEW

**Comment by cindyli** [comment]
Shall this branch add a grunt task for linting?

**Comment by jobara** [comment]
@cindyli this pull request is ready for another round of review.

File: provisioning/playbook.yml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+---
+- hosts: localhost
+  user: root
+
+  vars_files:
+    - vars.yml
+
+  roles:
+    - facts
+    - nodejs
```
Comment (cindyli) [inline provisioning/playbook.yml]: No newline at the end of the file, same as "provisioning/requirements.yml" and "provisioning/vars.yml".

**Comment by cindyli** [comment]
Indentations use 2 spaces in "Vagrantfile" and .yml files. Shall we use conventional 4 spaces?

**Comment by jobara** [comment]
I'm not sure. The Vagrant file is in ruby, and the others are yml, as opposed to JavaScript which is what our styles guidelines are for. I could change this for now, but we may want to come up with a specific style guide and linting for these file types in the future. We should talk this over with @avtar later because these files will likely come from another repo in the future.

**Comment by cindyli** [comment]
Merged @ ddc2dae77054a5d463027554ddc4b24bcafa4ae9

**Pull request merged**